resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -117,8 +118,19 @@ function task_install {
     install -m 0755 ext/ezbake-functions.sh "${DESTDIR}${app_prefix}"
     install -m 0644 ext/ezbake.manifest "${DESTDIR}${app_prefix}"
     install -d -m 0755 "${DESTDIR}${projconfdir}/conf.d"
+<% if EZBake::Config[:split_bootstraps] -%>
```

Comment:
seems worth putting this install call into the conditional below to keep them together.